packages/clay-color-picker/src/Custom.tsx

```diff
@@ -55,15 +75,11 @@ const RGBInput: React.FunctionComponent<IRGBInputProps> = ({
 					<ClayInput
 						data-testid={`${name}Input`}
 						insetBefore
-						onChange={(event) => {
-							const newVal = Number(event.target.value);
-
-							setInputValue(newVal);
-
-							onChange({[name]: newVal});
-						}}
+						max="255"
+						min="0"
+						onChange={handleOnChange}
 						ref={inputRef}
-						type="text"
+						type="number"
```

Comment:
Makes sense since this is going to accept (numeric) values between 0 and 255 :+1: